R/apply_col_plan.R

```diff
@@ -288,6 +302,18 @@ split_data_names_to_df <- function(data_names, preselected_cols, column_names){
 
 }
 
+#' Combines the split out data.frame of the potential columns and their spans into a named vector.
+#'
+#'
+#' @param data_names original names of the dataset
```

Comment:
this should be @param split_data_names + however it should be described since it is a whole tibble of info

Reply:
the input is variable depending on the column name structure, so its a bit hard to explain the specific data. but I can try to add more information.